### ISSUE TITLE
Output of Softmax layer give wrong result ?

### ISSUE THREAD

**Issue by theiron97** [keras-team/keras#10820]
I have applied softmax layer for output of RNN. But when I show sum of probability, I see some sum not equal 1:
![keras1](https://user-images.githubusercontent.com/15943606/43516042-8328ca60-95ae-11e8-9ea7-33e9cd24dbc3.png)
